cmd/swarm/swarm-smoke/feed_upload_and_sync.go

```diff
@@ -284,14 +307,87 @@ func cliFeedUploadAndSync(c *cli.Context) error {
 }
 
 func fetchFeed(topic string, user string, endpoint string, original []byte, ruid string) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ctx, sp := spancontext.StartSpan(ctx, "feed-and-sync.fetch")
+	defer sp.Finish()
+
 	log.Trace("sleeping", "ruid", ruid)
 	time.Sleep(3 * time.Second)
 
 	log.Trace("http get request (feed)", "ruid", ruid, "api", endpoint, "topic", topic, "user", user)
-	res, err := http.Get(endpoint + "/bzz-feed:/?topic=" + topic + "&user=" + user)
+
+	tn := time.Now()
+	reqUri := endpoint + "/bzz-feed:/?topic=" + topic + "&user=" + user
+	req, _ := http.NewRequest("GET", reqUri, nil)
+
+	opentracing.GlobalTracer().Inject(
+		sp.Context(),
+		opentracing.HTTPHeaders,
+		opentracing.HTTPHeadersCarrier(req.Header))
+
+	trace := &httptrace.ClientTrace{
+		GetConn: func(_ string) {
+			log.Trace("http get request (feed) - GetConn")
+			metrics.GetOrRegisterResettingTimer("feed-and-sync.fetch.clienttrace.getconn", nil).Update(time.Since(tn))
+		},
+		GotConn: func(_ httptrace.GotConnInfo) {
+			log.Trace("http get request (feed) - GotConn")
```

Comment:
@justelad please add `ruid`s to every trace, as we have concurrent goroutine requests.